src/schema/metadata/DatasetDOI.yaml

```diff
@@ -0,0 +1,10 @@
+---
+key_name: DatasetDOI
+description: |
+  The Digital Object Identifier of the dataset (not the corresponding paper).
+  DOIs SHOULD be expressed as a valid
+  [URI](/02-common-principles.html#uniform-resource-indicator);
+  bare DOIs such as `10.0.2.3/dfjj.10` are
+  [DEPRECATED](/02-common-principles.html#definitions).
```

Comment:
I started replacing internal relative links (e.g., `../02-common-principles.html#definitions`) with absolute versions (`/02-common-principles.html#definitions`). How does everyone feel about this change? I think it would be good to apply throughout the whole specification.

Reply:
Why are you doing that? I thought it's an advantage to point to the relative files and have Mkdocs generate the suitable links for us? Pointing to html directly feels error prone but I can't put my finger on it right now :thinking:

Reply:
One reason is that relative links depend on where the metadata field is rendered, so what might work for one file might not work for another. It should also make it easier to resolve links in external applications (e.g., NIDM-Terms).

Reply:
I see :thinking: I couldn't find info on whether that's good or bad: https://www.mkdocs.org/user-guide/writing-your-docs/#internal-links

it's not really mentioned in the docs over there. So maybe we should carefully try it, and if it doesn't break anything it's fine.